packages/pyright-internal/src/tests/harness/fourslash/testState.ts

```diff
@@ -924,7 +925,12 @@ export class TestState {
         }
     }
 
-    verifyHover(kind: MarkupKind, map: { [marker: string]: string | null }): void {
+    verifyHoverBase<T>(
```

Comment:
```suggestion
    private _verifyHoverBase<T>(
```
(note that eslint will probably make you move this method if we make it private)